vizro-core/src/vizro/static/css/layout.css

```diff
@@ -131,32 +131,28 @@ div.dashboard_container .tooltip-inner {
 div.dashboard_container .custom-tooltip {
   color: var(--text-primary);
 }
-.icon-text {
-  font-size: var(--text-size-02);
-  letter-spacing: var(--letter-spacing-body-edit-01);
-  color: var(--text-secondary);
-  padding: 4px;
-}
-
-.nav-icon {
-  filter: var(--fill-accordion-button);
-  width: 24px;
-  height: 24px;
-}
-
-.nav-icon-text {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  gap: 6px;
-  justify-content: center;
-}
 
-.icon-button.btn.btn-primary.active {
-  background-color: var(--state-overlays-selected);
+.icon-button.btn.btn-primary.active .material-symbols-outlined {
+  color: var(--text-active);
 }
-
 .loading-container {
   height: 100%;
   width: 100%;
 }
+
+.mantine-Tooltip-tooltip {
+  font-family: Inter;
+  font-size: var(--text-size-01);
+  font-weight: var(--text-weight-light);
+  line-height: 16px;
+  letter-spacing: var(--letter-spacing-help-text);
```

Comment:
Do you actually have to set all of these? I would have thought that some of them such as font-family etc. are not required anymore because they are defined in our global CSS variables